[debug] [2025-07-15T00:49:30.678Z] ----------------------------------------------------------------------
[debug] [2025-07-15T00:49:30.685Z] Command:       C:\Program Files\nodejs\node.exe C:\Users\THINKPAD\AppData\Roaming\npm\node_modules\firebase-tools\lib\bin\firebase.js deploy
[debug] [2025-07-15T00:49:30.686Z] CLI Version:   14.8.0
[debug] [2025-07-15T00:49:30.686Z] Platform:      win32
[debug] [2025-07-15T00:49:30.687Z] Node Version:  v22.13.1
[debug] [2025-07-15T00:49:30.688Z] Time:          Mon Jul 14 2025 19:49:30 GMT-0500 (hora estándar de Colombia)
[debug] [2025-07-15T00:49:30.688Z] ----------------------------------------------------------------------
[debug] 
[debug] [2025-07-15T00:49:30.707Z] >>> [apiv2][query] GET https://firebase-public.firebaseio.com/cli.json [none]
[debug] [2025-07-15T00:49:32.633Z] > command requires scopes: ["email","openid","https://www.googleapis.com/auth/cloudplatformprojects.readonly","https://www.googleapis.com/auth/firebase","https://www.googleapis.com/auth/cloud-platform"]
[debug] [2025-07-15T00:49:32.634Z] > authorizing via signed-in user (abollanedapedraza@gmail.com)
[debug] [2025-07-15T00:49:32.635Z] [iam] checking project techstore-c9c56 for permissions ["datastore.indexes.create","datastore.indexes.delete","datastore.indexes.list","datastore.indexes.update","firebase.projects.get","firebasehosting.sites.update"]
[debug] [2025-07-15T00:49:32.637Z] Checked if tokens are valid: true, expires at: 1752542371742
[debug] [2025-07-15T00:49:32.637Z] Checked if tokens are valid: true, expires at: 1752542371742
[debug] [2025-07-15T00:49:32.638Z] >>> [apiv2][query] POST https://cloudresourcemanager.googleapis.com/v1/projects/techstore-c9c56:testIamPermissions [none]
[debug] [2025-07-15T00:49:32.638Z] >>> [apiv2][(partial)header] POST https://cloudresourcemanager.googleapis.com/v1/projects/techstore-c9c56:testIamPermissions x-goog-quota-user=projects/techstore-c9c56
[debug] [2025-07-15T00:49:32.638Z] >>> [apiv2][body] POST https://cloudresourcemanager.googleapis.com/v1/projects/techstore-c9c56:testIamPermissions {"permissions":["datastore.indexes.create","datastore.indexes.delete","datastore.indexes.list","datastore.indexes.update","firebase.projects.get","firebasehosting.sites.update"]}
[debug] [2025-07-15T00:49:43.067Z] *** [apiv2] error from fetch(https://firebase-public.firebaseio.com/cli.json, {"headers":{},"method":"GET"}): FetchError: request to https://firebase-public.firebaseio.com/cli.json failed, reason: getaddrinfo ENOTFOUND firebase-public.firebaseio.com
[warn] !  Unable to fetch the CLI MOTD and remote config. This is not a fatal error, but may indicate an issue with your network connection. 
[debug] [2025-07-15T00:49:43.069Z] Failed to fetch MOTD FirebaseError: Failed to make request to https://firebase-public.firebaseio.com/cli.json
[debug] [2025-07-15T00:49:44.674Z] *** [apiv2] error from fetch(https://cloudresourcemanager.googleapis.com/v1/projects/techstore-c9c56:testIamPermissions, {"headers":{},"method":"POST","body":"{\"permissions\":[\"datastore.indexes.create\",\"datastore.indexes.delete\",\"datastore.indexes.list\",\"datastore.indexes.update\",\"firebase.projects.get\",\"firebasehosting.sites.update\"]}"}): FetchError: request to https://cloudresourcemanager.googleapis.com/v1/projects/techstore-c9c56:testIamPermissions failed, reason: getaddrinfo ENOTFOUND cloudresourcemanager.googleapis.com
[debug] [2025-07-15T00:49:44.675Z] [iam] error while checking permissions, command may fail: Failed to make request to https://cloudresourcemanager.googleapis.com/v1/projects/techstore-c9c56:testIamPermissions
[debug] [2025-07-15T00:49:44.676Z] Checked if tokens are valid: true, expires at: 1752542371742
[debug] [2025-07-15T00:49:44.677Z] Checked if tokens are valid: true, expires at: 1752542371742
[debug] [2025-07-15T00:49:44.677Z] >>> [apiv2][query] GET https://firebase.googleapis.com/v1beta1/projects/techstore-c9c56 [none]
[debug] [2025-07-15T00:49:44.685Z] *** [apiv2] error from fetch(https://firebase.googleapis.com/v1beta1/projects/techstore-c9c56, {"headers":{},"method":"GET","signal":{}}): FetchError: request to https://firebase.googleapis.com/v1beta1/projects/techstore-c9c56 failed, reason: getaddrinfo ENOTFOUND firebase.googleapis.com
[debug] [2025-07-15T00:49:44.685Z] Failed to make request to https://firebase.googleapis.com/v1beta1/projects/techstore-c9c56 (original: request to https://firebase.googleapis.com/v1beta1/projects/techstore-c9c56 failed, reason: getaddrinfo ENOTFOUND firebase.googleapis.com)
[error] 
[error] Error: Assertion failed: resolving hosting target of a site with no site name or target name. This should have caused an error earlier
[debug] [2025-07-15T01:26:43.369Z] ----------------------------------------------------------------------
[debug] [2025-07-15T01:26:43.385Z] Command:       C:\Program Files\nodejs\node.exe C:\Users\THINKPAD\AppData\Roaming\npm\node_modules\firebase-tools\lib\bin\firebase.js deploy
[debug] [2025-07-15T01:26:43.386Z] CLI Version:   14.8.0
[debug] [2025-07-15T01:26:43.386Z] Platform:      win32
[debug] [2025-07-15T01:26:43.386Z] Node Version:  v22.13.1
[debug] [2025-07-15T01:26:43.387Z] Time:          Mon Jul 14 2025 20:26:43 GMT-0500 (hora estándar de Colombia)
[debug] [2025-07-15T01:26:43.388Z] ----------------------------------------------------------------------
[debug] 
[debug] [2025-07-15T01:26:43.394Z] >>> [apiv2][query] GET https://firebase-public.firebaseio.com/cli.json [none]
[debug] [2025-07-15T01:26:44.033Z] > command requires scopes: ["email","openid","https://www.googleapis.com/auth/cloudplatformprojects.readonly","https://www.googleapis.com/auth/firebase","https://www.googleapis.com/auth/cloud-platform"]
[debug] [2025-07-15T01:26:44.034Z] > authorizing via signed-in user (abollanedapedraza@gmail.com)
[debug] [2025-07-15T01:26:44.034Z] [iam] checking project techstore-c9c56 for permissions ["datastore.indexes.create","datastore.indexes.delete","datastore.indexes.list","datastore.indexes.update","firebase.projects.get","firebasehosting.sites.update"]
[debug] [2025-07-15T01:26:44.036Z] Checked if tokens are valid: false, expires at: 1752542371742
[debug] [2025-07-15T01:26:44.036Z] Checked if tokens are valid: false, expires at: 1752542371742
[debug] [2025-07-15T01:26:44.036Z] > refreshing access token with scopes: []
[debug] [2025-07-15T01:26:44.038Z] >>> [apiv2][query] POST https://www.googleapis.com/oauth2/v3/token [none]
[debug] [2025-07-15T01:26:44.038Z] >>> [apiv2][body] POST https://www.googleapis.com/oauth2/v3/token [omitted]
[debug] [2025-07-15T01:26:55.579Z] *** [apiv2] error from fetch(https://firebase-public.firebaseio.com/cli.json, {"headers":{},"method":"GET"}): FetchError: request to https://firebase-public.firebaseio.com/cli.json failed, reason: getaddrinfo ENOTFOUND firebase-public.firebaseio.com
[warn] !  Unable to fetch the CLI MOTD and remote config. This is not a fatal error, but may indicate an issue with your network connection. 
[debug] [2025-07-15T01:26:55.581Z] Failed to fetch MOTD FirebaseError: Failed to make request to https://firebase-public.firebaseio.com/cli.json
[debug] [2025-07-15T01:26:56.081Z] *** [apiv2] error from fetch(https://www.googleapis.com/oauth2/v3/token, {"headers":{},"method":"POST","body":{"_overheadLength":545,"_valueLength":212,"_valuesToMeasure":[],"writable":false,"readable":true,"dataSize":0,"maxDataSize":2097152,"pauseStreams":true,"_released":true,"_streams":[],"_currentStream":null,"_insideLoop":false,"_pendingNext":false,"_boundary":"--------------------------655828215017435100553246","_events":{},"_eventsCount":1}}): FetchError: request to https://www.googleapis.com/oauth2/v3/token failed, reason: getaddrinfo ENOTFOUND www.googleapis.com
[error] Authentication Error: Your credentials are no longer valid. Please run firebase login --reauth

For CI servers and headless environments, generate a new token with firebase login:ci
[debug] [2025-07-15T01:26:56.082Z] [iam] error while checking permissions, command may fail: Authentication Error: Your credentials are no longer valid. Please run firebase login --reauth

For CI servers and headless environments, generate a new token with firebase login:ci
[debug] [2025-07-15T01:26:56.084Z] Checked if tokens are valid: false, expires at: 1752542371742
[debug] [2025-07-15T01:26:56.084Z] Checked if tokens are valid: false, expires at: 1752542371742
[debug] [2025-07-15T01:26:56.084Z] > refreshing access token with scopes: []
[debug] [2025-07-15T01:26:56.085Z] >>> [apiv2][query] POST https://www.googleapis.com/oauth2/v3/token [none]
[debug] [2025-07-15T01:26:56.085Z] >>> [apiv2][body] POST https://www.googleapis.com/oauth2/v3/token [omitted]
[debug] [2025-07-15T01:26:56.090Z] *** [apiv2] error from fetch(https://www.googleapis.com/oauth2/v3/token, {"headers":{},"method":"POST","body":{"_overheadLength":545,"_valueLength":212,"_valuesToMeasure":[],"writable":false,"readable":true,"dataSize":0,"maxDataSize":2097152,"pauseStreams":true,"_released":true,"_streams":[],"_currentStream":null,"_insideLoop":false,"_pendingNext":false,"_boundary":"--------------------------395845928755410559906031","_events":{},"_eventsCount":1}}): FetchError: request to https://www.googleapis.com/oauth2/v3/token failed, reason: getaddrinfo ENOTFOUND www.googleapis.com
[error] Authentication Error: Your credentials are no longer valid. Please run firebase login --reauth

For CI servers and headless environments, generate a new token with firebase login:ci
[debug] [2025-07-15T01:26:56.091Z] Authentication Error: Your credentials are no longer valid. Please run firebase login --reauth

For CI servers and headless environments, generate a new token with firebase login:ci
[error] 
[error] Error: Assertion failed: resolving hosting target of a site with no site name or target name. This should have caused an error earlier
[debug] [2025-07-15T01:27:49.779Z] ----------------------------------------------------------------------
[debug] [2025-07-15T01:27:49.785Z] Command:       C:\Program Files\nodejs\node.exe C:\Users\THINKPAD\AppData\Roaming\npm\node_modules\firebase-tools\lib\bin\firebase.js login
[debug] [2025-07-15T01:27:49.786Z] CLI Version:   14.8.0
[debug] [2025-07-15T01:27:49.787Z] Platform:      win32
[debug] [2025-07-15T01:27:49.787Z] Node Version:  v22.13.1
[debug] [2025-07-15T01:27:49.787Z] Time:          Mon Jul 14 2025 20:27:49 GMT-0500 (hora estándar de Colombia)
[debug] [2025-07-15T01:27:49.788Z] ----------------------------------------------------------------------
[debug] 
[debug] [2025-07-15T01:27:49.793Z] >>> [apiv2][query] GET https://firebase-public.firebaseio.com/cli.json [none]
[info] Already logged in as abollanedapedraza@gmail.com
[debug] [2025-07-15T01:28:00.623Z] Error: Timed out.
    at Timeout._onTimeout (C:\Users\THINKPAD\AppData\Roaming\npm\node_modules\firebase-tools\lib\utils.js:267:49)
    at listOnTimeout (node:internal/timers:594:17)
    at process.processTimers (node:internal/timers:529:7)
[error] 
[error] Error: An unexpected error has occurred.
[debug] [2025-07-15T01:28:20.600Z] ----------------------------------------------------------------------
[debug] [2025-07-15T01:28:20.607Z] Command:       C:\Program Files\nodejs\node.exe C:\Users\THINKPAD\AppData\Roaming\npm\node_modules\firebase-tools\lib\bin\firebase.js deploy
[debug] [2025-07-15T01:28:20.608Z] CLI Version:   14.8.0
[debug] [2025-07-15T01:28:20.608Z] Platform:      win32
[debug] [2025-07-15T01:28:20.609Z] Node Version:  v22.13.1
[debug] [2025-07-15T01:28:20.610Z] Time:          Mon Jul 14 2025 20:28:20 GMT-0500 (hora estándar de Colombia)
[debug] [2025-07-15T01:28:20.610Z] ----------------------------------------------------------------------
[debug] 
[debug] [2025-07-15T01:28:20.615Z] >>> [apiv2][query] GET https://firebase-public.firebaseio.com/cli.json [none]
[debug] [2025-07-15T01:28:21.469Z] > command requires scopes: ["email","openid","https://www.googleapis.com/auth/cloudplatformprojects.readonly","https://www.googleapis.com/auth/firebase","https://www.googleapis.com/auth/cloud-platform"]
[debug] [2025-07-15T01:28:21.470Z] > authorizing via signed-in user (abollanedapedraza@gmail.com)
[debug] [2025-07-15T01:28:21.471Z] [iam] checking project techstore-c9c56 for permissions ["datastore.indexes.create","datastore.indexes.delete","datastore.indexes.list","datastore.indexes.update","firebase.projects.get","firebasehosting.sites.update"]
[debug] [2025-07-15T01:28:21.473Z] Checked if tokens are valid: false, expires at: 1752542371742
[debug] [2025-07-15T01:28:21.473Z] Checked if tokens are valid: false, expires at: 1752542371742
[debug] [2025-07-15T01:28:21.474Z] > refreshing access token with scopes: []
[debug] [2025-07-15T01:28:21.476Z] >>> [apiv2][query] POST https://www.googleapis.com/oauth2/v3/token [none]
[debug] [2025-07-15T01:28:21.476Z] >>> [apiv2][body] POST https://www.googleapis.com/oauth2/v3/token [omitted]
[debug] [2025-07-15T01:28:32.755Z] *** [apiv2] error from fetch(https://firebase-public.firebaseio.com/cli.json, {"headers":{},"method":"GET"}): FetchError: request to https://firebase-public.firebaseio.com/cli.json failed, reason: getaddrinfo ENOTFOUND firebase-public.firebaseio.com
[warn] !  Unable to fetch the CLI MOTD and remote config. This is not a fatal error, but may indicate an issue with your network connection. 
[debug] [2025-07-15T01:28:32.757Z] Failed to fetch MOTD FirebaseError: Failed to make request to https://firebase-public.firebaseio.com/cli.json
[debug] [2025-07-15T01:28:33.525Z] *** [apiv2] error from fetch(https://www.googleapis.com/oauth2/v3/token, {"headers":{},"method":"POST","body":{"_overheadLength":545,"_valueLength":212,"_valuesToMeasure":[],"writable":false,"readable":true,"dataSize":0,"maxDataSize":2097152,"pauseStreams":true,"_released":true,"_streams":[],"_currentStream":null,"_insideLoop":false,"_pendingNext":false,"_boundary":"--------------------------285082668095574183920602","_events":{},"_eventsCount":1}}): FetchError: request to https://www.googleapis.com/oauth2/v3/token failed, reason: getaddrinfo ENOTFOUND www.googleapis.com
[error] Authentication Error: Your credentials are no longer valid. Please run firebase login --reauth

For CI servers and headless environments, generate a new token with firebase login:ci
[debug] [2025-07-15T01:28:33.527Z] [iam] error while checking permissions, command may fail: Authentication Error: Your credentials are no longer valid. Please run firebase login --reauth

For CI servers and headless environments, generate a new token with firebase login:ci
[debug] [2025-07-15T01:28:33.528Z] Checked if tokens are valid: false, expires at: 1752542371742
[debug] [2025-07-15T01:28:33.528Z] Checked if tokens are valid: false, expires at: 1752542371742
[debug] [2025-07-15T01:28:33.529Z] > refreshing access token with scopes: []
[debug] [2025-07-15T01:28:33.529Z] >>> [apiv2][query] POST https://www.googleapis.com/oauth2/v3/token [none]
[debug] [2025-07-15T01:28:33.530Z] >>> [apiv2][body] POST https://www.googleapis.com/oauth2/v3/token [omitted]
[debug] [2025-07-15T01:28:33.534Z] *** [apiv2] error from fetch(https://www.googleapis.com/oauth2/v3/token, {"headers":{},"method":"POST","body":{"_overheadLength":545,"_valueLength":212,"_valuesToMeasure":[],"writable":false,"readable":true,"dataSize":0,"maxDataSize":2097152,"pauseStreams":true,"_released":true,"_streams":[],"_currentStream":null,"_insideLoop":false,"_pendingNext":false,"_boundary":"--------------------------934771782923502468730767","_events":{},"_eventsCount":1}}): FetchError: request to https://www.googleapis.com/oauth2/v3/token failed, reason: getaddrinfo ENOTFOUND www.googleapis.com
[error] Authentication Error: Your credentials are no longer valid. Please run firebase login --reauth

For CI servers and headless environments, generate a new token with firebase login:ci
[debug] [2025-07-15T01:28:33.536Z] Authentication Error: Your credentials are no longer valid. Please run firebase login --reauth

For CI servers and headless environments, generate a new token with firebase login:ci
[error] 
[error] Error: Assertion failed: resolving hosting target of a site with no site name or target name. This should have caused an error earlier
